sys-fs/dwarfs/dwarfs-0.12.3-r1.ebuild

```diff
@@ -60,12 +60,8 @@ RESTRICT="!test? ( test )"
 CHECKREQS_DISK_BUILD="500M"
 CMAKE_WARN_UNUSED_CLI=0
 
-src_prepare(){
-	cmake_src_prepare
-	sed "s/DESTINATION lib/DESTINATION $(get_libdir)/" -i cmake/libdwarfs.cmake || die
-}
-
 src_configure(){
+	use stacktrace && append-flags -g
```

Comment:
I assume the idea behind this is that configuring an exception formatter doesn't do anything if you don't build with DWARF debuginfo?

Reply:
And I see that cpptrace does support GNU debuglink, so this should work well with gentoo's `FEATURES="splitdebug"` (by default it will just strip all that info).

Reply:
> I assume the idea behind this is that configuring an exception formatter doesn't do anything if you don't build with DWARF debuginfo?

There will still be stack traces even without debug symbols, but they'll require post-processing to be meaningful as they'll only contain addresses.

> And I see that cpptrace does support GNU debuglink, so this should work well with gentoo's `FEATURES="splitdebug"` (by default it will just strip all that info).

Yes, I've actually tested that this works fine with `splitdebug`.